backend/common/utils/result_notification.py

```diff
@@ -24,7 +33,7 @@ def notify_slack(data: dict):
 def format_failed_batch_issue_slack_alert(data: dict) -> dict:
     aws_region = os.getenv("AWS_DEFAULT_REGION")
     job_id = os.getenv("AWS_BATCH_JOB_ID")
-    job_url = f"https://{aws_region}.console.aws.amazon.com/batch/v2/home?region={aws_region}#jobs/detail/{job_id}"
+    job_url = aws_batch_job_url_fmt_str.format(aws_region=aws_region, job_id=job_id)
```

Comment:
I'm not sure I see the utility in creating str templates given they're only used once, but I guess it doesn't really hurt

Reply:
it was mainly to reducing cognitive load in that function. The long format string caused `black` to reformat it up in a way that was more confusing to understand.